ARKitExample/AppDelegate.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  AppDelegate.swift
+//  ARKitExample
+//
+//  Created by Lizzie Siegle on 8/10/17.
```

Comment:
File header for the quickstarts shouldn't have the author line.

Reply:
Normally we do not, I was wondering if we should keep this considering Lizzie originally wrote the example and she isn't a part of the iOS SDK team.